prow/tide/tide.go

```diff
@@ -1604,7 +1610,7 @@ func (c *syncController) presubmitsForBatch(prs []CodeReviewCommon, org, repo, b
 
 	var result []config.Presubmit
 	for _, ps := range presubmits {
-		if !ps.ContextRequired() {
+		if !c.provider.jobIsRequiredByTide(&ps, &prs[0]) {
```

Comment:
Nit: It's not clear to me why the 1st PR (`prs[0]`) is special. Add a comment?

Reply:
done